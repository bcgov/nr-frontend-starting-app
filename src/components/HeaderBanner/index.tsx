import styled from 'styled-components';

const HeaderBanner = styled.div`
  display: flex;
  justify-content: flex-start;
  align-items: center;
  margin: 0 10px 0 0;
`;

export default HeaderBanner;
