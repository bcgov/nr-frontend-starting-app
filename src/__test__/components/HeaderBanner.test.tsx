/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import HeaderBanner from '../../components/HeaderBanner';

// This component serves only as a 'parent' for the header title and image
// so the snapshot test is sufficient

describe('the component renders correctly', () => {
  it('should match the snapshot', () => {
    const tree = renderer
      .create(<HeaderBanner />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
