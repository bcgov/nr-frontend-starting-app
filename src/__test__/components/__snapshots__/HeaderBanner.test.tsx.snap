// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the component renders correctly should match the snapshot 1`] = `
<div
  className="sc-bczRLJ jtcqdn"
/>
`;
